--- upx-3.03-src/src/pefile.cpp.orig	2008-04-27 17:03:00.000000000 +0200
+++ upx-3.03-src/src/pefile.cpp	2009-08-28 16:02:31.000000000 +0200
@@ -1349,8 +1349,8 @@
     // FIXME this comparison is not too exact
     while (1)
     {
-        char *delim1 = strchr(keep, '/');
-        char *delim2 = strchr(keep, ',');
+        const char *delim1 = strchr(keep, '/');
+        const char *delim2 = strchr(keep, ',');
         if (helper::match(itype, ntype, keep))
         {
             if (!delim1)
